tests/unit/lib/test_datachain.py

```diff
@@ -791,12 +792,26 @@ def test_from_csv_no_header_output_list(tmp_dir, catalog):
 def test_from_csv_tab_delimited(tmp_dir, catalog):
     df = pd.DataFrame(DF_DATA)
     path = tmp_dir / "test.csv"
-    df.to_csv(path, sep="\t")
+    df.to_csv(path, sep="\t", index=False)
     dc = DataChain.from_csv(path.as_uri(), delimiter="\t")
     df1 = dc.select("first_name", "age", "city").to_pandas()
     assert df1.equals(df)
 
 
+def test_from_csv_null_collect(tmp_dir, catalog):
+    # Clickhouse requires setting type to Nullable(Type).
+    # See https://github.com/xzkostyan/clickhouse-sqlalchemy/issues/189.
+    skip_if_not_sqlite()
```

Comment:
I'm not sure how committed we are to clickhouse, so I think it's best to skip this for now in clickhouse and create an issue in studio. It would be a bit painful to implement support since we would have to wrap sql types with `clickhouse_sqlalchemy.types.Nullable` but only in clickhouse and only if the type may contain null values. Since all the types live in the open-source codebase, it would be painful to introduce clickhouse-specific logic.